src/detectors/Ruby/RubyLanguageDetector.ts

```diff
@@ -0,0 +1,37 @@
+import { ILanguageDetector } from '../ILanguageDetector';
+import { injectable, inject } from 'inversify';
+import { LanguageAtPath, ProgrammingLanguage } from '../../model';
+import { IFileInspector } from '../../inspectors/IFileInspector';
+import { Types } from '../../types';
+import { fileNameRegExp, fileExtensionRegExp, sharedSubpath } from '../utils';
+import { uniq } from 'lodash';
+import * as nodePath from 'path';
+
+@injectable()
+export class RubyLanguageDetector implements ILanguageDetector {
+  private fileInspector: IFileInspector;
+  constructor(@inject(Types.IFileInspector) fileInspector: IFileInspector) {
+    this.fileInspector = fileInspector;
+  }
+
+  async detectLanguage(): Promise<LanguageAtPath[]> {
+    const result: LanguageAtPath[] = [];
+    const packageFiles = await this.fileInspector.scanFor(fileExtensionRegExp(['gemspec']), '/');
```

Comment:
`gemspec` is not correct package manager definition file. It should be `Gemfile`. :)